src/routes/contact-us/+page.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+	import fb_logo from './Facebook_Logo_Primary.png';
+	import ig_logo from './Instagram_Glyph_Gradient.png';
+</script>
+
+<h1>Contact Us</h1>
+
+<p>
+	Email Address: <a href="mailto:kendoinfo@umich.edu"> kendoinfo@umich.edu</a>
+</p>
+
+<h2>Find us on:</h2>
+
+<figure>
```

Comment:
I don't think that this is a good use case for figure.

> Usually a `<figure>` is an image, illustration, diagram, code snippet, etc., that is referenced in the main flow of a document, but that can be moved to another part of the document or to an appendix without affecting the main flow.

source: [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#usage_notes)

Reply:
fixed in [c9e40ab](https://github.com/kendoclub-at-umich/website/pull/32/commits/c9e40ab31df496364a59d6205e91d6257d3e8094)